### PR TITLE
Fixed media controls responsiveness

### PR DESCRIPTION
**Pull request by kcarcia**
Addresses #1962 

Edit | Delete always stay on one line regardless of the browser window size.

### CODE REVIEW

File: app/views/shared/_media_objects_view.html.erb [hunk 1]
```diff
@@ -7,6 +7,7 @@
         <th style="max-width:10%">File</th>
         <th style="max-width:60%">Name</th>
         <th style="max-width:10%"></th>
+        <th style="max-width:10%"></th>
```
Comment (jaypoulz) [inline app/views/shared/_media_objects_view.html.erb]: This line is extra now

(6 headers, 5 tds)

Comment (kcarcia) [inline app/views/shared/_media_objects_view.html.erb]: Oops, tanks!

Comment (kcarcia) [inline app/views/shared/_media_objects_view.html.erb]: Actually @jaypoulz I did add another td, you sure this isn't okay? I'm counting 5 tds and 5 ths

Comment (jaypoulz) [inline app/views/shared/_media_objects_view.html.erb]: You're right, but the bottom code is a little wonk, so I'll add a few notes

File: app/views/shared/_media_objects_view.html.erb [hunk 2]
```diff
@@ -7,6 +7,7 @@
...
         <% if can_edit?( @project ) %>
```
Comment (jaypoulz) [inline app/views/shared/_media_objects_view.html.erb]: This guy doesn't have a corresponding td

File: app/views/shared/_media_objects_view.html.erb [hunk 3]
```diff
@@ -25,15 +26,15 @@
...
                 <% end %>
+            </td>
+            <td style="padding-left: 0px;">
                 <% if can_delete? mo %>
-                  | <%= link_to "Delete", mo, method: 'delete', class: "media_delete", data: { confirm: "Are you sure?" } %>
+                &nbsp;|&nbsp;<%= link_to "Delete", mo, method: 'delete', class: "media_delete", data: { confirm: "Are you sure?" } %>
                 <% end %>
-              </div>
             </td>
             <% if ["Project","News","Tutorial", "Visualization"].include? object.class.name%>
```
Comment (jaypoulz) [inline app/views/shared/_media_objects_view.html.erb]: This ``"if"`` is more inclusive than the one in line 14. Seems a little funny to me.

**Comment by kcarcia** [comment]
@jaypoulz I think this is better. Think. I mean that Display column I'm pretty sure wasn't doing anything. It's the featured image column we care about.

**Comment by kcarcia** [comment]
@jaypoulz Think I fixed the weirdness that was going on.

**Pull request merged**